### PR TITLE
NI geographies - lookups & postcode

### PR DESCRIPTION
**Pull request by BobHarper1**
At the moment, because GN relies on Code Point open for both postcode and ward/district/area lookups, NI geocodes and names are missing. I've added the relevant NI lookup csv, and modified the data import to add these codes and names, meaning any grant data with an geocode in NI will have the additional fields added.

Also, if  a postcode starts with 'BT' we can safely assume this is in Northern Ireland (a simple fact, there's no licensing issues in this case).

### CODE REVIEW

File: LICENCE.rst [hunk 1]
```diff
@@ -34,7 +34,14 @@ Contains OS data © Crown copyright and database right 2016
...
 
-Contains National Statistics data © Crown copyright and database right 2016  
+Contains National Statistics data © Crown copyright and database right 2016
+
+Northern Ireland Ward to Council Area Lookup
+---------------
+
+GrantNav uses `Northern Ireland Ward to Council Area Lookup <https://ons.maps.arcgis.com/home/item.html?id=cce0999ed17f4fbd9f2f5480997405c5>`_ data which is distributed under an `Open Government Licence (v3) <http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/>`_
+
+Contains National Statistics data © Crown copyright and database right 2015
```
Comment (Bjwebb) [inline LICENCE.rst]: Copyright info also needs adding to:
* The datasets page: https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/templates/datasets.html#L10 
* The CSV template: https://github.com/OpenDataServices/grantnav/blob/master/grantnav/csv_layout.py#L68

Comment (BobHarper1) [reply LICENCE.rst]: Thanks Ben, I've added data source. Publisher was National Statistics so added 2015 copyright year to their attribution.

Comment (Bjwebb) [reply LICENCE.rst]: Thanks, licensing info looks good now.

**Comment by BobHarper1** [comment]
Thanks Ben. Quick look at results:

| **current.acceptable-license-valid.ni-geography** |   | 
|------------------|--------------|
| Total grants     | 9,504        |
| Total funders    | 22           |
| Total recipients | 5,781        |
| Total awarded    | £413,468,227 |
|  |  |
| **live site grantnav.threesixtygiving.org** |   | 
| Total grants     | 9,398        |
| Total funders    | 18           |
| Total recipients | 5,694        |
| Total awarded    | £410,992,934 |

**Pull request merged**